ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/retriever/RecoveringSidecarRetriever.java

```diff
@@ -90,10 +111,13 @@ public synchronized void start() {
     }
     cancellable =
         asyncRunner.runWithFixedDelay(
-            this::checkPendingPromises,
-            recoverInitiationCheckInterval,
-            recoverInitiationCheckInterval,
-            error -> LOG.error("Failed to check pending Sidecar retrievals", error));
+            this::checkPendingRequests,
+            recoveryInitiationCheckInterval,
+            recoveryInitiationCheckInterval,
+            error ->
+                LOG.error(
+                    "Failed to check pending data column sidecar requests for recovery initiation",
```

Comment:
can we get some context into this message in case we need to try to figure this out from client logs?